src/components/Page/components/Header/Header.tsx

```diff
@@ -228,6 +214,42 @@ export function Header({
   );
 }
 
+function PrimaryActionMarkup({
+  primaryAction,
+}: {
+  primaryAction: PrimaryAction | React.ReactNode;
+}) {
+  const {isNavigationCollapsed} = useMediaQuery();
+  const {newDesignLanguage} = useFeatures();
```

Comment:
Curious if anyone thinks we should pass these in as props?

Reply:
Yeah I was curious about this too.

Reply:
I'd leave it. It will be easier to clean-up and really doesn't become part of the API 🤷